package.json

```diff
@@ -58,14 +57,16 @@
     "prettier": "^2.4.1",
     "prettier-plugin-import-sort": "^0.0.7",
     "typedoc": "^0.22.6",
-    "typescript": "^4.4.4"
+    "typescript": "^4.4.4",
+    "whatwg-fetch": "^3.6.2"
```

Comment:
is this needed?

Reply:
yes, for `fetch-intercept` to work during the `test` phase